mediabridge/data_processing/wiki_to_netflix.py

```diff
@@ -4,12 +4,15 @@
 import time
 from tqdm import tqdm
 import sys
+from db.insert_data import insert_into_mongo
+from constants import *
```

Comment:
The use of `import *` is an anti-pattern. It results in a bunch of random 'magical' variables being in your script, with readers of the script having no way of knowing where they came from. For instance, if someone sees `DATA_DIR` below and looks at the imports, they would have to assume it comes from this import. But what if you had multiple `import *` statements? The variable could be coming from any of them.

Prefer `from constants import DATA_DIR, OUT_DIR` or `import constants` and then using `constants.DATA_DIR` below.

Reply:
That's a good point, just updated :)